infra/envs/test/main.tf

```
@@ -10,36 +10,68 @@ terraform {
   backend "s3" {
     encrypt = true
     region  = "us-east-1"
-    profile = "czi-hca-dev"
+    profile = "single-cell-dev"
   }
 }
 
 provider "aws" {
   version = "~> 2.0"
   region  = "us-east-1"
-  profile = "czi-hca-dev"
+  profile = "single-cell-dev"
 }
 
-module "ledger" {
-  source = "../../modules/backend/ledger"
+//module "ledger" {
+//  source = "../../modules/backend/ledger"
+//
+//  deployment_stage = "${var.deployment_stage}"
+//
+//  // Database
+//  db_username                  = "${var.db_username}"
+//  db_password                  = "${var.db_password}"
+//  db_instance_count            = "${var.db_instance_count}"
+//  preferred_maintenance_window = "${var.preferred_maintenance_window}"
+//}
 
-  deployment_stage = "${var.deployment_stage}"
+//module "browser" {
+//  source = "../../modules/backend/browser"
+//
+//  deployment_stage = "${var.deployment_stage}"
+//
+//  // Database
+//  db_username                  = "${var.browser_db_username}"
+//  db_password                  = "${var.browser_db_password}"
+//  db_instance_count            = "${var.browser_db_instance_count}"
+//  preferred_maintenance_window = "${var.browser_preferred_maintenance_window}"
+//}
 
-  // Database
-  db_username                  = "${var.db_username}"
-  db_password                  = "${var.db_password}"
-  db_instance_count            = "${var.db_instance_count}"
-  preferred_maintenance_window = "${var.preferred_maintenance_window}"
+module "siteCert" {
+  source = "github.com/chanzuckerberg/cztack//aws-acm-cert?ref=v0.29.0"
+
+  # the cert domain name
+  cert_domain_name = "browser.dev.single-cell.czi.technology"
+
```

I _think_ what is going on is that the static site is also trying to set up the www. hostname. 

Try adding 
```suggestion
cert_subject_alternative_names = {"www.browser.dev.single-cell.czi.technology": "Z0921546EDJ5WWGRFFKB"}
```

---

Thank you this fixed it